apps/web/src/middlewares/cors.ts

```diff
@@ -3,7 +3,7 @@ import Cors from "cors"
 import { type NextApiHandler, type NextApiRequest, type NextApiResponse } from "next"
 
 const cors = Cors({
-  origin: ["http://localhost:3002"],
+  origin: ["http://localhost:3002", "http://dev.dashboard.online.ntnu.no", "https://dev.dashboard.online.ntnu.no"],
```

Comment:
https
```suggestion
  origin: ["http://localhost:3002", "https://dev.dashboard.online.ntnu.no", "https://dev.dashboard.online.ntnu.no"],
```

Reply:
I added both just out of lazyness, but i remove the http instance now